src/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -226,10 +227,14 @@ export const OuiBreadcrumbs: FunctionComponent<OuiBreadcrumbsProps> = ({
       className: breadcrumbClassName,
       ...breadcrumbRest
     } = breadcrumb;
-
+    let lastBreadCrumbColor: OuiLinkColor = 'subdued';
```

Comment:
Can this be rewritten to not use mutable lets? I.e. using a function or (preferably) using a ternary operator?